### PR TITLE
fix sidebar contribution link

### LINKED ISSUE #530: `Contribute to Docs` link is broken
`Contribute to Docs` link is broken.
helm-www

It should probably be `https://github.com/helm/helm-www/tree/master/content/en/docs`.
`/en` might be missing.

### PR DESCRIPTION
**Pull request by lemonli**
Signed-off-by: lemonli <liwenjun0323@gmail.com>

fix sidebar `Contribute to Docs` link

fix #530

### CODE REVIEW

File: themes/helm/layouts/partials/sidebar.html [hunk 1]
```diff
@@ -41,7 +41,7 @@
 
     <aside class="sidebar-buttons is-hidden-mobile">
       <div class="button-wrap">
-        <a href="https://github.com/helm/helm-www/tree/master/content/docs" class="button is-outline is-fullwidth">Contribute to Docs</a>
+        <a href="https://github.com/helm/helm-www/blob/master/CONTRIBUTING.md" class="button is-outline is-fullwidth">Contribute to Docs</a>
```
Comment (hickeyma) [inline themes/helm/layouts/partials/sidebar.html]: The link might better be served as: https://github.com/helm/helm-www#contributing

Comment (lemonli) [reply themes/helm/layouts/partials/sidebar.html]: ok, I will update this link at once

Comment (lemonli) [reply themes/helm/layouts/partials/sidebar.html]: Completed

Comment (lemonli) [reply themes/helm/layouts/partials/sidebar.html]: sorry, I did not saw this issue [#530](https://github.com/helm/helm-www/issues/530) before submit this pr

**Pull request merged**